core/src/main/java/org/apache/calcite/sql/dialect/SnowflakeSqlDialect.java

```diff
@@ -134,6 +129,9 @@ private void unparseOtherFunction(SqlWriter writer, SqlCall call, int leftPrec,
       call.operand(0).unparse(writer, leftPrec, rightPrec);
       writer.endFunCall(formatDate);
       break;
+    case "IF":
+      unparseIf(writer, call, leftPrec, rightPrec);
+      break;
```

Comment:
is this case different from IF case above, if not remove else merge both cases

Reply:
done